### PR TITLE
Capes Support - Initial commits from @navarr

### PR DESCRIPTION
**Pull request by Jake0oo0**
Cape endpoint, with major networking changes to support caching and retrieving of multiple uuid-related elements. 

- [x] Caching
- [x] Networking Changes
- [x] Downloading capes
- [x] Documentation

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -4,10 +4,17 @@ var logger = require("morgan");
 var cookieParser = require("cookie-parser");
 var bodyParser = require("body-parser");
 
+<<<<<<< HEAD
```
Comment (jomo) [inline app.js]: :+1:

**Comment by Jake0oo0** [comment]
:shipit:

**Comment by Jake0oo0** [comment]
IT PASSES!

**Comment by Jake0oo0** [comment]
give it some thorough testing/review @jomo thanks g'night :crescent_moon:

**Comment by jomo** [comment]
Will do soon

**Comment by Jake0oo0** [comment]
50 commits later..I _think_ it works. Can you give it some testing @jomo?

**Comment by Jake0oo0** [comment]
Going to benchmark this on the new server tonight and hopefully release it along with the new server in production. @jomo

**Comment by jomo** [comment]
Could you also do #74 to make sure we have a log file somewhere?
thanks to my horrible ISP I've been offline for a week and they don't seem to be making any progress. I only have internet access at work or via my (horrible) phone connection at home.
I'll try to get on IRC later today though.

**Pull request merged**